spec/features/admin_can_view_items_list_spec.rb

```diff
@@ -8,7 +8,8 @@
       end
 
       admin = create(:admin_user)
-      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+      allow_any_instance_of(ApplicationController).to receive(:current_user)
+                                                  .and_return(admin)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 44) spaces for indenting an expression spanning multiple lines.